### PR TITLE
Angela patch3

### PR DESCRIPTION
**Pull request by afloresOSI**
Updates to the Quick Start topics to include descriptions for diagrams, updated diagrams, and ensure that procedures match the Configuration section.

### CODE REVIEW

File: V1/Overview/PIEgressQuickStart.md [hunk 1]
```diff
@@ -4,13 +4,24 @@ uid: piEgressQuickStart
 
 # PI egress quick start
 
-This topic provides quick start instructions for egressing data stored in the Edge Data Store into a remote PI System. This is accomplished using PI Web API which is configured for basic authentication.
+Data egress provides a mechanism to transfer data to PI Server using OMF messages through a PI Web API endpoint. To get started sending data stored in EDS to a PI System, create a PI Web API OMF endpoint and configure periodic egress to use the PI Web API endpoint.
+
+## Create a PI Web API OMF endpoint
+
+1. Install PI Web API and enable the **OSIsoft Message Format (OMF) Services** feature.
```
Comment (TinaUll) [inline V1/Overview/PIEgressQuickStart.md:11]: What if PI Web API is already installed for some reason?

File: V1/Overview/PIEgressQuickStart.md [hunk 2]
```diff
@@ -4,13 +4,24 @@ uid: piEgressQuickStart
...
+    - During configuration, choose an AF database and PI Data Archive where metadata and data will be stored.
+    - The account used in an egress configuration needs permissions to create AF elements, element templates, and PI points.
+2. Configure PI Web API to use *Basic* authentication.
+
+ For complete steps, as well as best practices and recommendations, see the [PI Web API User Guide](https://livelibrary.osisoft.com/LiveLibrary/web/pub.xql?c=t&action=home&pub=web-api-v12&lang=en#addHistory=true&filename=GUID-D49D9A09-B47E-4FF7-A9D3-B7C1FEA5E681.xml&docid=GUID-D807EF71-7F37-43DB-A357-EF03CCD001F1&inner_id=&tid=&query=&scope=&resource=&toc=false&eventType=lcContent.loadDocGUID-D807EF71-7F37-43DB-A357-EF03CCD001F1) on Live Library.
```
Comment (TinaUll) [inline V1/Overview/PIEgressQuickStart.md:16]: There are "friendlier" URLs available on LL: https://livelibrary.osisoft.com/LiveLibrary/content/en/web-api-v12/GUID-D807EF71-7F37-43DB-A357-EF03CCD001F1

That link is hidden under the Share topic button on the right side.

File: V1/Overview/SDSQuickStart.md [hunk 3]
```diff
@@ -75,7 +83,7 @@ Complete the following steps to create an SDS stream.
 
 ## Write data events to the SDS stream
 
-After you have created a type and container, you can write data using SDS. Complete the following steps to write data to a stream.
+After creating a type and container, use SDS to write data to a stream.
```
Comment (TinaUll) [inline V1/Overview/SDSQuickStart.md:86]: Another idea for this sentence that avoids the gerund?

File: V1/Overview/OCSEgressQuickStart.md [hunk 4]
```diff
@@ -4,11 +4,23 @@ uid: ocsEgressQuickStart
 
 # OCS egress quick start
 
-This topic provides quick start instructions for getting data stored in the Edge Data Store into OCS. You can accomplish this by using the OCS OMF endpoint which is configured for OCS authentication.
+Data egress provides a mechanism to transfer data to OCS using OMF messages. To get started sending data stored in EDS to OSIsoft Cloud Services, create an OMF connection in OCS and configure an egress endpoint with the connection information for OCS.
```
[the remark this answers is not in the document]
Comment (afloresOSI) [reply V1/Overview/OCSEgressQuickStart.md:7]: If I do that, I would want to add a link to the full configuration section to each quick start for consistency.

File: V1/Overview/OCSEgressQuickStart.md [hunk 5]
```diff
@@ -33,16 +45,16 @@ This topic provides quick start instructions for getting data stored in the Edge
...
+2. Modify the **Endpoint** parameter to be the URL of your OCS OMF endpoint.
+3. Modify the **ClientId** and **ClientSecret** parameters with the authentication information to connect to your OCS OMF endpoint.
 
-    **Note:** If required, use the StreamPrefix and TypePrefix to ensure uniqueness on the destination system. If a StreamPrefix is specified, it is used to create a unique stream id on OCS. This configuration is set up to send all stream data to OCS. If you want to only send specific streams, edit the EgressFilter value. For examples of more advanced scenarios, see [Data egress configuration](xref:egress).
+    **Note:** If required, use the **StreamPrefix** and **TypePrefix** parameters to ensure uniqueness on the destination system. If a StreamPrefix is specified, it is used to create a unique stream ID on OCS. This configuration is set up to send all stream data to OCS. To only send specific streams, edit the **EgressFilter** parameter. For examples of more advanced scenarios, see [Data egress configuration](xref:egress).
```
[the remark this answers is not in the document]
Comment (afloresOSI) [reply V1/Overview/OCSEgressQuickStart.md]: Reworked the sentence.

File: V1/Overview/OpcUaQuickStart.md [hunk 6]
```diff
@@ -4,15 +4,19 @@ uid: opcUaQuickStart
 
 # OPC UA EDS adapter quick start
 
-This topic provides quick start instructions for setting up an OPC UA EDS adapter. You can add a single OPC UA EDS adapter during Edge Data Store installation. For more information, see [Install Edge Data Store](xref:InstallEdgeDataStore).  
+ The OPC UA EDS adapter is a component of Edge Data Store that connects to OPC UA capable devices. The OPC UA EDS adapter can be configured to have multiple instances within EDS. Each instance of the adapter can connect to one device. The EDS installation includes the OPC UA EDS adapter and the option to add a single OPC UA EDS adapter instance. Additional instances can be added after installation. For more information about installation, see [Install Edge Data Store](xref:InstallEdgeDataStore). To get started collecting data with an instance of the OPC UA EDS adapter, you need to configure the data source, which specifies the device connection, and the data selection, which specifies the data to collect.
```
[the remark this answers is not in the document]
Comment (afloresOSI) [reply V1/Overview/OpcUaQuickStart.md:7]: I want to make sure that it is clear that the adapter is only installed once, but that you define additional instances (OPCUA1, OPCUA2, etc.).

File: V1/Overview/OMFQuickStart.md [hunk 7]
```diff
@@ -40,11 +48,11 @@ The first step in OMF data ingress is to create an OMF type that describes the f
...
+When this command completes successfully, an OMF type with the same name is created on the server. Any number of containers can be created from the type, as long as they use a timestamp as an index and have a 32-bit floating point value. The Type definition needs to be sent first before containers and data, but it does not cause an error if the same definition is sent at a later time.
 
 ## Create an OMF container
 
-The next step in writing OMF data is to create a container. As with an OMF type, this only needs to be done once before sending data events, and resending the same definition repeatedly does not cause an error.
+The next step in writing OMF data is to create a container. As with an OMF type, this only needs to be sent once before sending data events, but resending the same definition repeatedly does not cause an error.
```
[the remark this answers is not in the document]
Comment (afloresOSI) [reply V1/Overview/OMFQuickStart.md]: Clarified that I am talking about sending OMF messages.

**Pull request merged**